instances/treasury-devdao.near/widget/config/data.jsx

```diff
@@ -9,14 +9,14 @@ return {
       title: "Payments",
       href: "?page=payments",
     },
-    {
-      title: "Stake Delegation",
-      href: "?page=stake-delegation",
-    },
-    {
-      title: "Asset Exchange",
-      href: "?page=asset-exchange",
-    },
+    // {
```

Comment:
why commenting these? will they be enabled later?

Reply:
yes, we are starting testing with NF and since there pages aren't built yet, I have commented it.